### ISSUE TITLE
Spurious warnings for Password for the master DB user is readable

### ISSUE THREAD

**Issue by zapman449** [wata727/tflint#249]
```
$ tflint -v
TFLint version 0.7.2
```

When I run tflint, I get spurious errors about the master db user password being readable.  But it's not.

Warning message:
```
frig_rds.tf
	WARNING:64 Password for the master DB user is readable. recommend using environment variables. (aws_db_instance_readable_password)
```

Code snippit in question:
```
variable "frig_db_password" {}
# ...
resource "aws_db_instance" "frig_db" {
  count                     = "${lookup(var.frig_db_toggle_map, var.env_prefix, "${var.frig_db_toggle_default}")}"
  allocated_storage         = 100
  engine                    = "postgres"
  engine_version            = "9.5.10"
  parameter_group_name      = "default.postgres9.5"
  instance_class            = "${lookup(var.frig_db_class_map, var.env_prefix, "${var.frig_db_class_default}")}"
  storage_type              = "gp2"
  name                      = "frig_db"
  username                  = "root"
  password                  = "${var.frig_db_password}"
  port                      = 5432
  db_subnet_group_name      = "${aws_db_subnet_group.default.name}"
  multi_az                  = "${lookup(var.frig_db_multiaz_bool_map, var.env_prefix, "${var.frig_db_multiaz_bool_default}")}"
  publicly_accessible       = false
  storage_encrypted         = true
  apply_immediately         = true
  backup_retention_period   = 7
  backup_window             = "06:15-07:15"
  maintenance_window        = "Sun:09:05-Sun:10:05"
  skip_final_snapshot       = "false"
  final_snapshot_identifier = "frig-db-tf-final-snapshot"

  vpc_security_group_ids = [
    "${aws_security_group.rds_sg.id}",
  ]

  tags {
    Name        = "frig_db_tf"
    Owner       = "FRIG"
    TeamOwner   = "Frig"
    CostCenter  = "Shared-Cloud-Service"
    Environment = "${var.prefix_environment}"
    Role        = "Frig RDS"
  }
}
```

**Comment by wata727** [comment]
TFLint v0.8.0 has been released. Please try the latest version again.
